### DISCUSSION TITLE
fr/tasks/obb/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#19681] [Docs]
# fr/tasks/obb/

Découvrez comment détecter des objets avec rotation pour une plus grande précision à l'aide des modèles OBB de YOLO11 . Apprenez, formez, validez et exportez des modèles OBB sans effort.

https://docs.ultralytics.com/fr/tasks/obb/

**Comment by hariouat** [comment]
Hello, 
Please, is there any problem with the new version of ultralytics.? i'm trying to use obb for yolov11 but the bounding box for validation data are not drawing but for the training all it's okay. 
When i used i sample python code the annotations correct.

**Reply by glenn-jocher** [reply]
@hariouat thanks for reaching out! To visualize validation bounding boxes with YOLO11-OBB, please ensure you're using the latest `ultralytics` package and add the `save_hybrid=True` argument during validation. For example:

```python
model.val(data="your_data.yaml", save_hybrid=True)
```

This will save plotted predictions alongside ground truth boxes. If issues persist, please verify your validation dataset format matches the OBB requirements outlined in the Oriented Bounding Box (OBB) Datasets Guide.

**Comment by UltralyticsAssistant** [comment]
👋 Bonjour, merci pour votre intérêt pour Ultralytics 🚀! Nous vous recommandons de consulter les [Docs](https://docs.ultralytics.com/fr/) où vous trouverez des exemples pour [Python](https://docs.ultralytics.com/fr/usage/python/) et [CLI](https://docs.ultralytics.com/fr/usage/cli/), ainsi que des réponses aux questions les plus courantes.

Si ceci est un 🐛 Rapport de bug, veuillez fournir un [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) pour nous aider à résoudre le problème.

Si vous avez des ❓ Questions sur l'entraînement personnalisé, partagez un maximum d'informations, comme des exemples d'images de votre dataset et des logs d'entraînement. Assurez-vous de suivre nos [Conseils pour de meilleurs résultats](https://docs.ultralytics.com/fr/guides/model-training-tips/).

Rejoignez la communauté Ultralytics sur votre plateforme préférée : 
- Pour des discussions en temps réel, rendez-vous sur [Discord](https://discord.com/invite/ultralytics) 🎧
- Pour des discussions approfondies, explorez [Discourse](https://community.ultralytics.com/)
- Ou participez à des échanges sur notre [Subreddit](https://reddit.com/r/Ultralytics) pour partager vos connaissances.

## Mise à jour

Mettez à jour le package `ultralytics` avec toutes les [dépendances nécessaires](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) dans un environnement [**Python>=3.8**](https://www.python.org/) avec [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) pour vérifier si le problème est déjà résolu dans la dernière version :

```bash
pip install -U ultralytics
```

## Environnements

YOLO peut être exécuté dans les environnements vérifiés à jour suivants (avec toutes les dépendances, y compris [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) et [PyTorch](https://pytorch.org/) préinstallés) :

- **Notebooks** avec GPU gratuit : <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. Consultez le [Guide de démarrage GCP](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. Consultez le [Guide de démarrage AWS](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Image Docker**. Consultez le [Guide de démarrage Docker](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Statut

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

Si ce badge est vert, tous les tests [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) fonctionnent actuellement. Les tests CI vérifient le fonctionnement correct de tous les [Modes](https://docs.ultralytics.com/fr/modes/) et [Tâches](https://docs.ultralytics.com/fr/tasks/) YOLO sur macOS, Windows et Ubuntu toutes les 24 heures et à chaque commit.

Ceci est une réponse automatique 🤖. Un ingénieur Ultralytics vous assistera bientôt ! 😊
